doc/docs_es.md

```diff
@@ -0,0 +1,116 @@
+# Documentación de V
+
+## Introducción
+
+V es un lenguaje de programación compilado de tipo estático diseñado para crear software mantenible.
```

Comment:
`tipado estático` ?

Reply:
> Se dice de un lenguaje de programación que usa un tipado estático cuando la comprobación de tipificación se realiza durante la compilación, y no durante la ejecución. Ejemplos de lenguajes que usan tipado estático son C, C++, Java y Haskell.

Reply:
right, I meant that it should be `**tipado** estático`, not `**tipo** estático` :)

Reply:
Oh sorry xD

Reply:
Fixed up :)